Fixing build with gcc-4.5

http://bugs.gentoo.org/show_bug.cgi?id=317163

Following patch was applied in 2.13 branch
Thanks Patrick McCarty for information

--- lily/spacing-spanner.cc
+++ lily/spacing-spanner.cc
@@ -38,8 +38,8 @@
   vsize end = binary_search (all, (Grob*) me->get_bound (RIGHT),
 			     &Paper_column::less_than);  
   
-  all = vector<Grob*>::vector<Grob*> (all.begin () + start,
-				      all.begin () + end + 1);
+  all = vector<Grob*> (all.begin () + start,
+		       all.begin () + end + 1);
   return all;
 }
 
